weblab/core/tests/test_processing.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from django.conf import settings
+from core.processing import prepend_callback_base
+
+
+@pytest.mark.django_db
+def test_prepend_callback_base():
+    old_callback_base = settings.CALLBACK_BASE_URL
+    old_script_name = settings.FORCE_SCRIPT_NAME
+    settings.CALLBACK_BASE_URL = 'http://test.com/'
+    settings.FORCE_SCRIPT_NAME = None
```

Comment:
It's better to use the settings fixture for this, as it will automatically restore the old settings however the test exits. See https://pytest-django.readthedocs.io/en/latest/helpers.html#settings